index.html

```diff
@@ -4,9 +4,77 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/f8112dc1f2.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
   <title>My Portfolio</title>
 </head>
 <body>
-  
+  <header>
+    <nav>
+      <a class="wrap logo" href="#">Kehinde Audu</a>
+      <i class="fas fa-bars"></i>
+    </nav>
+  </header>
+  <main>
+    <div class="hero">
+      <h1>Kehinde Audu</h1>
+      <p class="tag">Software Developer</p>
+      <p class="decription">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like 
+        what you see and have a project you need coded,  don’t hestiate to contact me.
+      </p>
+      <div class="abilities">
+        <div class="languages wrap">
+          <p>Languages</p>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div class="frameworks wrap">
+          <p>Frameworks</p>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="skills wrap">
+          <p>Skills</p>
+          <ul>
+            <li>Codekit</li>
+            <li>GitHub</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div>
+      <img class="elipse" src="img/elipse.svg.svg" alt="">
```

Comment:
Please do not use the `img` tag for decoration images. They should be background images.